projects/plugins/crm/.gitignore

```diff
@@ -19,4 +20,5 @@
 /templates/quotes/.htaccess
 /templates/quotes/index.html
 /tests/acceptance.suite.yml
```

Comment:
```suggestion
/tests/acceptance.suite.yml
/tests/codeception/acceptance.suite.yml
```

Reply:
Nice catch @cleacos 👏
I have added the new path to .gitignore and a comment for the old path so we can remove it in the future if we want to keep the file clean